### PR TITLE
feat(settings): remove expert mode

### LINKED ISSUE #3904: [CoW Swap] Remove expert mode

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Closes #3904 

Removes expert mode completely

# To Test

1. All 3 flows (SWAP, LIMIT, TWAP)
2. Safe bundling
3. EthFlow
4. Classic EthFlow
* Everything should work as before
* There should be no expert mode setting

### CODE REVIEW

**Review by elena-zh — CHANGES_REQUESTED** [review]
Hey @alfetopito , great, thank you!
However, classic eth-flow is not working here: no button to wrap/approve. Actually, this behavior corresponds to the expert mode (no buttons). 
image

And this should work this way: 
image

All the rest changes LGTM!

**Comment by alfetopito** [comment]
> Hey @alfetopito , great, thank you! However, classic eth-flow is not working here: no button to wrap/approve. Actually, this behavior corresponds to the expert mode (no buttons). image
> 
> And this should work this way: image
> 
> All the rest changes LGTM!

Should be fixed 🤞

File: apps/cowswap-frontend/src/modules/limitOrders/containers/SettingsWidget/index.tsx [hunk 1]
```diff
@@ -17,55 +15,26 @@ import {
 export function SettingsWidget() {
   const settingsState = useAtomValue(limitOrdersSettingsAtom)
   const updateSettingsState = useSetAtom(updateLimitOrdersSettingsAtom)
-  const [showExpertConfirm, setShowExpertConfirm] = useState(false)
 
   const onStateChanged = useCallback(
```
Comment (shoom3301) [inline apps/cowswap-frontend/src/modules/limitOrders/containers/SettingsWidget/index.tsx:19]: `onStateChanged` can be just replaced by `updateSettingsState`

**Review by shoom3301 — APPROVED** [review]
Awesome!
Haven't tested deeply, just a checked there is no option in the settings

**Review by elena-zh — APPROVED** [review]
Great job, @alfetopito !
I have found one low prio issue https://github.com/cowprotocol/cowswap/issues/3926 while testing the PR, but it is reproducible on Prod as well

File: apps/cowswap-frontend/src/modules/limitOrders/containers/SettingsWidget/index.tsx [hunk 2]
```diff
@@ -1,71 +1,29 @@
...
 
   return (
     <>
       <Menu>
         <SettingsButton>
           <SettingsIcon />
-          {settingsState.expertMode && (
-            <ExpertModeIndicator>
-              <span>🐮</span>
-              <span>🥋</span>
```
Comment (anxolin) [inline apps/cowswap-frontend/src/modules/limitOrders/containers/SettingsWidget/index.tsx:54]: Nooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo


Kimono CoW! This is a sad day

**Pull request merged**